src/components/MainContent/Content.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { ContentProps } from './Components';
+
+const SplitByNewLine = (text: string) => {
+  return text.split('\r\n');
+};
+
+const IsUnderlined = (line: string) => {
+  return (
+    line ===
+    '*(**Note**: In this automated version of the item specification, underlined text appears in italics.)*'
+  );
+};
+
+export const NewLine = ({ children }: ContentProps) => (
+  <span>
+    {children}
+    <br />
+  </span>
+);
+
+const ReplaceDashWithDot = (text: string) => text.replace('-   ', '• ');
+
+const ParseSingle = (text: string, underlined: boolean) => {
+  const parts = text.split('*');
+
+  return parts.map((part, index) => {
+    const noDashPart = ReplaceDashWithDot(part);
+    if (index % 2 === 1) {
+      if (underlined) return <u key={index}>{noDashPart}</u>;
+
+      return <i key={index}>{noDashPart}</i>;
+    }
+
+    return <React.Fragment key={index}>{noDashPart}</React.Fragment>;
+  });
+};
+
+const ParseContent = (text: string, underlined: boolean) => {
+  const parts = text.split('**');
+
+  return parts.map((part, index) => {
+    const parsedLine = ParseSingle(part, underlined);
+    if (index % 2 === 1) {
+      return <strong key={index}>{parsedLine}</strong>;
+    }
+
+    return <React.Fragment key={index}>{parsedLine}</React.Fragment>;
+  });
+};
+
+export const Content = (text: string) => {
```

Comment:
Camel-case this one too.

Reply:
I think this is a component, not a general function.
Please let me know if I am missunderstanding the difference.

Reply:
Oh, my mistake. It is a function which just renders JSX elements.
Since the name also belongs to that of components, I ll change it as function's name